server/sonar-server/src/main/java/org/sonar/server/user/ServerUserSession.java

```diff
@@ -131,6 +133,30 @@ public boolean isRoot() {
   }
 
   @Override
+  public boolean hasOrganizationPermission(String organizationUuid, String permission) {
+    if (permissionsByOrganizationUuid == null) {
+      permissionsByOrganizationUuid = HashMultimap.create();
+    }
+    Set<String> permissions;
+    if (permissionsByOrganizationUuid.containsKey(organizationUuid)) {
```

Comment:
you can avoid calling both `containsKey` and `get` (they essentially do the same thing):
```java
Set<String> permissions = permissionsByOrganizationUuid.get(organizationUuid);
if (permissions == null) {
   permissions = loadOrganizationPermissions(organizationUuid);
   permissionsByOrganizationUuid.putAll(organizationUuid, permissions);
}
return permissions.contains(permission);
```